headers/functions/arm9.h

```diff
@@ -974,6 +974,7 @@ bool BaseFormsEqual(enum monster_id monster1, enum monster_id monster2);
 bool DexNumbersEqual(enum monster_id monster1, enum monster_id monster2);
 bool GendersEqual(enum monster_id monster1, enum monster_id monster2);
 bool GendersEqualNotGenderless(enum monster_id monster1, enum monster_id monster2);
+bool GendersOppositeNotGenderless(enum monster_id monster1, enum monster_id monster2);
```

Comment:
Can we call this GendersNotEqualNotGenderless for consistency with GendersEqualNotGenderless?